### PR TITLE
Create config dashboard component

### LINKED ISSUE #174: [Feature Request] - Create config dashboard component (view only)
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find another request for the same or similar feature.

## Is your feature request related to a problem, or an open issue? Describe the problem, and if applicable, reference the issue.

The UI dashboard for admins should be updated to include a new component which retrieves all config items from the DB and displays them.

## Describe the solution you'd like

A new component on the dashboard should...
- List all config items
- Refresh the items on a recurring basis (every 10s)

### PR DESCRIPTION
**Pull request by Peruggia29**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

Added new component to the dashboard. Along with the component, we added two forms boxes titled Team Registration and Support Request. 

Worked with @smitshah2531 on this issue.

## Related Issues

Closes #174 - Creating dashboard component

### CODE REVIEW

**Comment by johnkahn** [comment]
@SpencerKaiser I worked with them yesterday on this and this might need to be broken into different issues, because there were some questions around how would updating them work with it overwriting the inputs every 10 seconds. So currently this is just displaying them, and then my thought was to have an edit button that would pop a modal or something that isn't overwritten by the 10 second fetch. But that solution seemed a bit much so I told them to hold off until I could ask your opinion.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -64,7 +70,7 @@ const AdminPage: NextComponentType = () => {
       clearInterval(timer);
     };
   }, []);
-
+
```
Comment (SpencerKaiser) [inline src/pages/index.tsx:73]:
```suggestion

```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -271,6 +293,32 @@ const AdminPage: NextComponentType = () => {
...
+                <div className="form-group ml-sm-3 mr-3 mb-2">
+                <label htmlFor={configItem.key} className="mr-3">
+                  {configItem.key}
+                </label>
+                      <input
+                        type="text"
+                        className="form-control"
+                        id={configItem.key}
+                        placeholder="Team name"
+                        value={configItem.value}
```
Comment (SpencerKaiser) [inline src/pages/index.tsx:314]:
```suggestion
                        value={configItem.value}
                        disabled
```

File: src/pages/index.tsx [hunk 3]
```diff
@@ -271,6 +293,33 @@ const AdminPage: NextComponentType = () => {
           </div>
         </div>
       </div>
+      <div className="row mt-3">
+      <div className="col-12 col-md-6">
+      <div className="card">
+            <div className="card-body">
+              <h2 className="font-weight-normal">Config Items</h2>
+              {results.length === 0 && <div className="alert alert-info mt-3">Display Database Config Items 🤔</div>}
```
Comment (SpencerKaiser) [inline src/pages/index.tsx:301]:
```suggestion
              {results.length === 0 && <div className="alert alert-info mt-3">No config items to display 🤔</div>}
```

**Review by SpencerKaiser — APPROVED** [review]

**Pull request merged**